### ISSUE TITLE
Proposal: add EnvVar block into basic template of helm create

### ISSUE THREAD

**Issue by itsmethemojo** [helm/helm#11284]
One of the first things i have to manually add when quickly using a default helm chart created with helm create is adding an configuration block adding environment variables

I prefer to store all them in a secret and embed them from secret within the deployment. This is especially a good idea when there are sensitive information in it.

So why add this functionality to the default template in blow up the binary with it? 

I have seen a dozen different implementations in custom charts throughout the years. All copied from somewhere having different flaws, like not using secrets and storing sensitive values in plain sight. If this is a out-of-the-box feature of a default chart, most developers wouldn't add this feature by stack-overflow copy-paste by themselves and a lot of secrets will stay as secret as they should be.

If you thing this might be as a good idea as i think i will create the PR for that.

I think it will be going in here https://github.com/helm/helm/blob/d7b4c38c42cb0b77f1bcebf9bb4ae7695a10da0b/pkg/chartutil/create.go#L514

**Comment by joejulian** [comment]
I'm sure folks would be happy to take a look at a PR for this.

**Comment by itsmethemojo** [comment]
here it is https://github.com/helm/helm/pull/11349

let me know when i forgot something basic. this is my first PR here

**Comment by itsmethemojo** [comment]
@joejulian is there a ready for review label that needs to be set?

**Comment by itsmethemojo** [comment]
@joejulian PR is ready for a while now. anything i missed in the progress?

**Comment by joejulian** [comment]
No, you're good. It needs to be reviewed by a core maintainer. I've added it to a milestone so it'll either get reviewed by that milestone deadline, or they'll bump it to the next one. Thanks for your contribution!

**Comment by itsmethemojo** [comment]
So i updated the MR a few weeks ago but received no feedback from any core mantainer at all. Looks like nobody wants this. Actually i would be happier to receive a "No we dont need that" then nothing
